### PR TITLE
Lab Cat caption added

### PR DESCRIPTION
**Pull request by sito-otis**
Changes made on the caption to resolve PR #9

### CODE REVIEW

**Review by mcbass1 — APPROVED** [review]
Looks good!

File: _slides/18-sito-otis.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Lab Cat! All for Science! Hurrah!
```
Comment (mcbass1) [inline _slides/18-sito-otis.md:6]: Looks good

**Review by mcbass1 — APPROVED** [review]
Looks good!

**Comment by sito-otis** [comment]
merged with main

**Pull request merged**